docs/deprecation-policy.rst

```diff
@@ -0,0 +1,23 @@
+====================================
+API stability and deprecation policy
+====================================
+
+1. ``urllib3`` will not wait until a "major" version release (e.g., 3.0.0,
```

Comment:
```suggestion
1. urllib3 will not wait until a "major" version release (e.g., 3.0.0,
```

First item can be "urllib3 doesn't follow Semantic Versioning".

(And use bullets, not numbers: https://asrfs.github.io/Tech-Writing-for-Techies/output/html/02-03-lists-instructions.html#_bullet_or_numbered)